src/js/setup-text-tracks.js

```diff
@@ -0,0 +1,144 @@
+import dashjs from 'dashjs';
+
+/*
+ * Attach text tracks from dash.js to videojs
+ *
+ * @param {videojs} player the videojs player instance
+ * @param {array} tracks the tracks loaded by dash.js to attach to videojs
+ *
+ * @private
+ */
+function attachDashTextTracksToVideojs(player, tech, tracks) {
+  const trackDictionary = [];
```

Comment:
Minor, but may want to change the name since it is an array. Maybe something like trackPairs?